src/KafkaFlow.Admin.Dashboard/AngularFiles/src/app/http-error.interceptor.ts

```diff
@@ -0,0 +1,31 @@
+import {
+    HttpEvent,
+    HttpInterceptor,
+    HttpHandler,
+    HttpRequest,
+    HttpResponse,
+    HttpErrorResponse
+   } from '@angular/common/http';
+   import { Observable, throwError } from 'rxjs';
+   import { retry, catchError } from 'rxjs/operators';
+   
+   export class HttpErrorInterceptor implements HttpInterceptor {
+    intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+      return next.handle(request)
+        .pipe(
+          retry(1),
+          catchError((error: HttpErrorResponse) => {
+            let errorMessage = '';
+            if (error.error instanceof ErrorEvent) {
+              // client-side error
+              errorMessage = `Error: ${error.error.message}`;
+            } else {
+              // server-side error
+              errorMessage = `Error Code: ${error.status}\nMessage: ${error.message}`;
+            }
+            window.alert(errorMessage);
```

Comment:
Show the message error in a notification component just like the success messages